packages/cli/templates/vue3/src/style.css

```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap');
 
```

Comment:
```
html {
  font-size: 62.5%;
}
```

If we set font-size to 62.5%, then it will be easier for developers to copy-paste styles from the Storyblok app into their plugins, because they won't have to recalculate the font size to a different rem basis.

We discussed this previously here: https://github.com/storyblok/field-plugin/pull/85

Reply:
I also replied there: https://github.com/storyblok/field-plugin/pull/85#discussion_r1143623870

This `font-size: 62.5%` is very specific to how Storyblok is designed currently, not it's not a common practice in general. I don't think we should add this to the template because commonly 1rem is 16px.

If users want to have Storyblok-like look & feel, they'd better use MUI instead of manually copy&paste css from browser inspector. If they use MUI, then font-size: 62.5% will automatically be applied.

So I think we should stay vanilla in this template.

Reply:
I agree with @eunjae-lee lets leave this one out, if the users want it they can add it simply.